tests/test_submain.py

```diff
@@ -291,7 +315,10 @@ def get_tests(self, tier, type):
 
         self.last_tier = tier
 
-        for x in range(1,10): # Ten times because we care
+        for x in range(1, 10):  # Ten times because we care
+            # ^ Very witty. However, it would be nice to actually know
+            # exactly why we're yielding these ten times.
+
```

Comment:
I guess what I'm more worried about is that it'll stop breaking something that we need it to break...